controllers/cloud.redhat.com/providers/cronjob/impl.go

```diff
@@ -170,6 +170,7 @@ func applyCronJob(app *crd.ClowdApp, env *crd.ClowdEnvironment, cj *batch.CronJo
 
 	utils.UpdateAnnotations(pt, common.KubeLinterAnnotations)
 	utils.UpdateAnnotations(cj, common.KubeLinterAnnotations)
+	utils.UpdateAnnotations(&cj.ObjectMeta, app.ObjectMeta.Annotations)
```

Comment:
Why is it not just update `cj` instead of `&cj.ObjectMeta`?

Reply:
I opened a PR that would allow you to do
`utils.UpdateAnnotations(cj, common.KubeLinterAnnotations, app.ObjectMeta.Annotations)`
Which feels nice to me

Reply:
Didn't know I could. Couldn't find an implementation for the GetAnnotations method that UpdateAnnotations called so I didn't know what it was going to edit. Changed in next push.

Reply:
I reviewed and approved that PR (thanks!) so if you merge it I'll rebase on it and update this PR.